cfgov/v1/auth_forms.py

```diff
@@ -36,14 +36,6 @@ class UserEditValidationMixin(PasswordValidationMixin):
     user_attribute = 'instance'
 
 
-class CFGOVUserCreationForm(UserEditValidationMixin, UserCreationForm):
-    pass
```

Comment:
yea this creates more technical debt, and future bugs with user creation which i'd rather avoid. I think its important to enforce the email uniqueness.